### ISSUE TITLE
Conformer yaml is missing several required hparams

### ISSUE THREAD

**Issue by mschonwe** [speechbrain/speechbrain#648]
When running the `recipes/LibriSpeech/ASR/transformer/train.py` with the `conformer.yaml` the training errors with several missing hparam errors.  Adding the following to the yaml allows training to proceed:

`skip_prep`, `tokenizer`, `pretrainer`, `stage_one_epochs`, `Adam` (renamed from optimizer)

Can submit as a PR if you want.

**Comment by TParcollet** [comment]
@JianyuanZhong

**Comment by mschonwe** [comment]
Also, some issues when doing eval with the recipe yaml:
might be an incorrect case of **T**ransformer for default `conformer.yaml` has:
```
modules:
  Transformer: *id003 
```
Which seems to run on training, but when running an eval with `TransformerASR.from_hparams` errors out.

Also... had to add modules: `compute_features`, `normalize`, `pre_transformer`, `beam_searcher` to get the eval to run.  Might be a good idea to have these in the default yaml.

**Comment by TParcollet** [comment]
Could you open a PR?

**Comment by TParcollet** [comment]
Right, let's see if we shouldn't simply remove this YAML for now. THe recipe isn't respecting the correct pretraining loading scheme and might induce peoples in the wrong direction.

**Comment by mschonwe** [comment]
Sorry about not responding re: the PR.  I'm not confident about what the _correct_ config should be, and don't want to introduce new issues.  

Running a training now with a YAML pieced together from the github version, and the https://huggingface.co/speechbrain/asr-transformer-transformerlm-librispeech/blob/main/hyperparams.yaml version to get training/inference to run.  (Getting long gibberish inference results so far, coming up on end of epoch 1, with ~16k hours of training data.)

**Comment by TParcollet** [comment]
Wow, that's a lot of training data. Note: You might want to work with the transformer rather than the conformer as transformers are way more stable (and faster) in the current version.

**Comment by mravanelli** [comment]
In general, the conformer part is still in beta version. We are working on that, but still not stable. @JianyuanZhong @popcornell

**Comment by TParcollet** [comment]
Being fixed in #535.
